slobrok/src/vespa/slobrok/server/remote_slobrok.h

```diff
@@ -67,9 +71,11 @@ class RemoteSlobrok: public IRpcServerManager,
     bool isConnected() const { return (_remote != nullptr); }
     void tryConnect();
     void maybePushMine();
+    void maybeStartFetch();
     void invokeAsync(FRT_RPCRequest *req, double timeout, FRT_IRequestWait *rwaiter);
     const std::string & getName() const { return _rpcserver.getName(); }
     const std::string & getSpec() const { return _rpcserver.getSpec(); }
+    const MapView &remoteView() const { return _mapView; }
```

Comment:
we should probably return MapSource here, which raises the question about mutable event listener lists...